backend/api/v1/issue_service.go

```diff
@@ -352,6 +343,33 @@ func (s *IssueService) SearchIssues(ctx context.Context, request *v1pb.SearchIss
 			} else {
 				issueFind.CreatedTsBefore = &ts
 			}
+		case "type":
+			if spec.operator != comparatorTypeEqual {
+				return nil, status.Errorf(codes.InvalidArgument, `only support "=" operation for "level" filter`)
+			}
+			if spec.value == "DDL" {
```

Comment:
In the new CI/CD, the issue type will be "v1pb.Issue_DATABASE_CHANGE", and we use the following to determine a change is for DDL/DML/NewDB. Is also challenging that existing pipelines don't have plans and new issues may not have rolllouts (with plan only). Maybe @RainbowDashy can help with details.

https://github.com/bytebase/bytebase/blob/abc705f033acfd4a25c7b54e74d69db9ca9022e5/proto/v1/rollout_service.proto#L240

https://github.com/bytebase/bytebase/blob/abc705f033acfd4a25c7b54e74d69db9ca9022e5/proto/v1/rollout_service.proto#L190C7-L190C27